app/bundles/CoreBundle/Test/Doctrine/DBALMocker.php

```diff
@@ -276,6 +277,10 @@ public function getMockResultStatement()
             ->method('fetch')
             ->willReturn($this->queryResponse);
 
+        $mock->expects($this->testCase->any())
```

Comment:
@escopecz I added this to the mock, I know the method to does not exists in the original class and this mocker needs update